cloudformation/PolicyTemplates/README.md

```diff
@@ -0,0 +1,24 @@
+## CIS Developer Policy Templates
```

Comment:
Is it cool to do this later?  I agree should be in read the docs ultimately.  Should this block merging?